src/js/apps/forms/form/form-history_app.js

```diff
@@ -0,0 +1,56 @@
+import Radio from 'backbone.radio';
+
+import App from 'js/base/app';
+
+import { LayoutView, HistoryBarView, FormView } from 'js/views/forms/form/form-history_views';
+
+export default App.extend({
+  viewEvents: {
+    'select:response': 'onSelectResponse',
+  },
+  stateEvents: {
+    'change:response': 'showIframeView',
+  },
+  onBeforeStart() {
+    this.showView(new LayoutView());
+    this.getRegion('iframe').startPreloader();
+  },
+  beforeStart({ action }) {
+    return action.fetch();
+  },
+  onStart({ form }, action) {
+    this.form = form;
+    const formResponses = action.getFormResponses();
+
+    const historyBarView = this.showChildView('history', new HistoryBarView({
+      formResponses,
+      state: {
+        selected: action.getFormResponses().first(),
```

Comment:
I think this and below should just be `formReponses.first()` since we got it a couple lines up.

But yeah I think this setting of the state is probably better rather than passing it since what `first` is could have changed from the fetch